addons/l10n_cl/models/res_partner.py

```diff
@@ -21,3 +22,34 @@ class ResPartner(models.Model):
              '2 - Fees Receipt Issuer (Applies to suppliers who issue fees receipt)\n'
              '3 - End consumer (only receipts)\n'
              '4 - Foreigner')
+
+    def _format_vat_cl(self, values):
+        identification_types = [self.env.ref('l10n_latam_base.it_vat').id, self.env.ref('l10n_cl.it_RUT').id,
+                                self.env.ref('l10n_cl.it_RUN').id]
+        partner_country_is_chile = (values.get('country_id') == self.env.ref('base.cl').id) or (
+                    values.get('l10n_latam_identification_type_id') and
+                    self.env['l10n_latam.identification.type'].browse(
+                        [values.get('l10n_latam_identification_type_id')]).country_id == self.env.ref('base.cl'))
```

Comment:
no need for [] as it is only a single id

Reply:
fixed fe5b839a6d2e4aab1248186384b45dbb56f0790b